templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -0,0 +1,62 @@
+mode: 0755
+path: "/usr/local/bin/configure-ovs.sh"
+contents:
+  inline: |
+    #!/usr/bin/env bash
```

Comment:
BTW after https://github.com/openshift/machine-config-operator/pull/1766 lands, it will become possible to add Go code into the MCD binary itself and have it execute on the host reliably.

Reply:
thanks. I had planned on moving to go after I got the bash working. I didn't know about this option.

Reply:
#1766 landed.  Now as I understand things this PR is about controlling the SDN, which we do *not* need during "firstboot" which is when we [perform os updates](https://github.com/openshift/machine-config-operator/blob/master/docs/OSUpgrades.md), only after kubelet is ready to start.

I think all of this bash code could become a `machine-config-daemon initialize-ovs` subcommand or so, then the MCD just invokes `systemd-run --unit mco-ovs /run/bin/machine-config-daemon initialize-ovs` when it starts up the first time on a node.  (Note that you will need to handle the case of the MCD being restarted without a node restart, e.g. new daemonset rollout, so your code should  be idempotent)

Reply:
OK after some real-time discussion, it sounds like we need this before kubelet, which does not fit neatly into the current architecture.  Today we have:

- Code shipped as part of the OS that runs before and up to kubelet (podman, crio, kubelet)
- The special MCD code introduced for #1766 - but that *only* applies on the firstboot, after that the MCD pod takes over (i.e. it's after kubelet)

(We're also implicitly having a larger debate about how much "host management" should happen in a pod)

One thing we could do is ship this code as part of a container image (actually, it could be the SDN container), and build upon the "host binary" trick we're already doing there, but instead of extracting it from the pod, have our systemd unit pull and extract it directly and run `Before=kubelet.service`.  Or, another variant of this is to basically just `podman run --privileged -v /:/host` and have the container perform the same trick the MCD does and extract its binary to the host, where it can then be run via `systemd-run`.